lib/support/Utils.ts

```diff
@@ -21,3 +23,29 @@ export function isObject(value: unknown): value is Record<string, any> {
 export function isFile(value: unknown): value is File {
   return value instanceof File
 }
+
+export function isYmd(value: unknown, required: YmdType[] = ['y', 'm', 'd']): value is Ymd {
+  if (!isObject(value)) {
+    return false
+  }
+
+  return required
+    .reduce<string[]>((keys, type) => {
+      keys.push(YmdMap[type])
+      return keys
+    }, [])
+    .every((key) => value[key] === null || isString(value[key]) || isNumber(value[key]))
```

Comment:
Why allow strings here? The type only has `number | null` 👀

Reply:
Ah, it's just a copy-paste issue.
`Ymd` and `Hms` has different types: `number | null` and `string | null`.
Fixed: 2a1bbd1